docs/docs/cells.md

```diff
@@ -143,14 +143,42 @@ export const beforeQuery = (props) => {
 
 For example, if you wanted to turn on Apollo's polling option, and prevent caching, you could export something like this (see Apollo's docs on [polling](https://www.apollographql.com/docs/react/data/queries/#polling) and [caching](https://www.apollographql.com/docs/react/data/queries/#setting-a-fetch-policy))
 
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+ 
 ```jsx
 export const beforeQuery = (props) => {
-  return { variables: props, fetchPolicy: 'no-cache', pollInterval: 2500 }
+  return {
+    variables: props,
```

Comment:
I added some line-breaks here to make things easier to read. (That were also missing originally)